src/librustdoc/clean/mod.rs

```diff
@@ -229,7 +229,9 @@ fn clean_generic_bound<'tcx>(
             GenericBound::TraitBound(clean_poly_trait_ref(t, cx), modifier)
         }
         // FIXME(precise_capturing): Implement rustdoc support
```

Comment:
```suggestion
```


Obv, RTN isn't fully supported with this PR but it's safe to say that this FIXME concerns the HIR only

Reply:
why didn't I remove this comment. I'm an idiot lol 🤦